src/libcommonserver/io/iohelper.cpp

```diff
@@ -284,8 +285,12 @@ bool IoHelper::getItemType(const SyncPath &path, ItemType &itemType) noexcept {
     const bool isSymlink = _isSymlink(path, ec);
 
     itemType.ioError = stdError2ioError(ec);
+#ifdef _WIN32
+    const bool fsSupportsSymlinks = Utility::isNtfs(path.parent_path());
```

Comment:
Use remove_filename() instead of parent_path() ?

Reply:
I should just provide `path` to `Utility::isNtfs`. In a first time, I used` parent_path()` because of `Utility:: isNtfs (const SyncPath &dirPath)`  definition which suggest it is expecting a directory. But as it internally only relies on `SyncPath::root_name()` there is no need to remove the file name.
Could be interesting to change `Utility:: isNtfs (const SyncPath &dirPath)` declaration.

Reply:
Here is the commit with the modifications: [b3028ae](https://github.com/Infomaniak/desktop-kDrive/pull/338/commits/b3028ae8c60abc4c121048dd2e4fc1f155486763)